src/services/posts/update-post.test.js

```diff
@@ -0,0 +1,58 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const updatePost = require('./update-post')
+
+
+describe('Update Article', () => {
+	let user
+	let article
+	let updatedInfo = {}
+
+	beforeEach(async () => {
```

Comment:
wondering if a `before` won't suffice here 

i might lack context but this is just a  call to evaluate if a `beforeEach` is absolutely integral to the test working as expected

Reply:
you are right